ruff.toml

```
@@ -26,6 +26,7 @@ extend-select = [
     "UP",
     "W",
     "YTT",
+    "PD",
 ]
 
 ignore = ["COM812", "FBT", "G004"]
```

Following from James' comments, I would exclude this from the whole codebase

```suggestion
ignore = ["COM812", "FBT", "G004", "PD901"]
```